File: go-sdk/pkg/models/operations/getapidisplayadsid.go
package operations

type GetAPIDisplayAdsIDPathParams struct {
	ID int32 `pathParam:"style=simple,explode=false,name=id"`
}

type GetAPIDisplayAdsIDRequest struct {
	PathParams GetAPIDisplayAdsIDPathParams
}

type GetAPIDisplayAdsIDResponse struct {
	ContentType string
	StatusCode  int64
}

File: ts-sdk/src/sdk/models/operations/getapidisplayadsid.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetApiDisplayAdsIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class GetApiDisplayAdsIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetApiDisplayAdsIdPathParams;
}


export class GetApiDisplayAdsIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
